src/test/java/scheduling_ct_use_case/ScheduleCTInteractorTest.java

```diff
@@ -0,0 +1,180 @@
+package scheduling_ct_use_case;
+
+import org.junit.jupiter.api.Test;
+import screens.collaborative_task_scheduling.*;
+import use_cases.collaborative_task_scheduling.scheduling_ct_use_case.*;
+import entities.*;
+
+import javax.swing.*;
+import java.awt.*;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.util.ArrayList;
+import java.util.HashMap;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.fail;
+
+class ScheduleCTInteractorTest {
+
+    // CHANGED USE CASE TO CALL TASK MAP INSTEAD OF BRINGING IN ALL TASKS
+    // NEED TO TEST DIFFERENTLY
```

Comment:
Just a suggestion since this is what I did: I added a BeforeAll method and manually set the TaskMap using TaskMap.setTaskMap()

Not sure if that's the best way to do things but it doesn't interfere with the data files so it should be ok